src/Altinn.Profile.Integrations/SblBridge/PartyNotificationContactPoints.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Altinn.Profile.Core.Unit.ContactPoints;
+using Altinn.Profile.Core.User.ContactPoints;
+
+namespace Altinn.Profile.Integrations.SblBridge
+{
+    /// <summary>
+    /// Model describing a container for a list of contact points.
+    /// </summary>
+    public class PartyNotificationContactPoints
+    {
+        /// <summary>
+        /// Gets or sets the party id of the organisation.
+        /// </summary>
+        public Guid? PartyId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the legacy id of the organisation.
+        /// </summary>
+        public int LegacyPartyId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the organization number of the organisation.
+        /// </summary>
+        public string OrganizationNumber { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets a list of multiple contanct points associated with the organisation.
+        /// </summary>
+        public List<UserRegisteredContactPoint> ContactPoints { get; set; } = new List<UserRegisteredContactPoint>();
+
+        /// <summary>
+        /// Maps a list of <see cref="PartyNotificationContactPoints"/> to a list of <see cref="UnitContactPoints"/>.
+        /// </summary>
+        public static UnitContactPointsList MapToUnitContactPoints(List<PartyNotificationContactPoints> source)
+        {
+            var contactPoints = source.Select(partyNotificationEndpoint => new UnitContactPoints
```

Comment:
Shouldn't we declare the type here for the variable `contactPoints`?

Reply:
I disagree. The property name is explicit enough the object is immediatly inserted into a UnitContactPointsList. Adding the explicit type here would only result in even more characters on the line.

Reply:
I see your point! Of course this would increase number of characters in this line. However, when I was reading this piece of code who had a little idea what is going on here, I wished that the type were here so I can get the context of this piece of code somewhat more comfortably. 🙂